src/index.ts

```diff
@@ -1302,6 +1316,31 @@ export default class Decimal {
       }
     }
 
+    //handle XFY format
```

Comment:
sure - looks like XpY but you put the two parts the other way around? (I also note it has different () handling to XfY but it'd be trivial to fix if it's a problem)

Reply:
Yeah, I figured parentheses would be useful to have for both parts